service/history/service.go

```diff
@@ -44,402 +42,17 @@ import (
 	"go.temporal.io/server/common/rpc"
 	"go.temporal.io/server/common/service/config"
 	"go.temporal.io/server/common/service/dynamicconfig"
-	"go.temporal.io/server/common/task"
+	"go.temporal.io/server/service/history/configs"
 )
 
-// Config represents configuration for history service
-type Config struct {
```

Comment:
I'm expecting all configs are moved as is without any changes to new package.

Reply:
all changes are made / to be made as is, or at least no real business logic change for the purpose of refactoring

Reply:
there can be go circular dependency, which requires some rewrite